autoreject/utils.py

```diff
@@ -15,6 +16,12 @@
 from mne.forward._field_interpolation import _setup_dots
 
 
+_GDKW = dict()
```

Comment:
for my knowledge, what does GDKW stand for? :)

Reply:
get data keyword arguments